packages/shared/src/components/comments/AdAsComment.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement, useContext, useEffect } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { Ad } from '../../graphql/posts';
+import { apiUrl } from '../../lib/config';
+import { TruncateText } from '../utilities';
+import AdLink from '../cards/AdLink';
+import { adLogEvent } from '../../lib/feed';
+import LogContext from '../../contexts/LogContext';
+import { ProfileImageSize, ProfilePicture } from '../ProfilePicture';
+import PlaceholderCommentList from './PlaceholderCommentList';
+
+export const AdAsComment = (): ReactElement => {
+  const { logEvent } = useContext(LogContext);
+  const ad = useQuery<Ad>(
+    ['ads', 'post'],
+    async () => {
+      const res = await fetch(`${apiUrl}/v1/a/post`);
+      const ads: Ad[] = await res.json();
+      return ads[0];
+    },
+    {
+      enabled: true,
+      refetchOnMount: false,
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+    },
+  );
+
+  const { isLoading, data, isSuccess, isError } = ad || {};
+  const { link, providerId, source, image, description, pixel } = data || {};
+
+  useEffect(() => {
```

Comment:
This one can track more then once technically, maybe you want to use ref for insurance of 1 impression tracked?

Reply:
Yup, on it